app/models/game.rb

```diff
@@ -0,0 +1,9 @@
+class Game < ApplicationRecord
+
+  belongs_to :match
+  has_one    :away_team, :through :matches
+  has_one    :home_team, :through :matches
+  has_one    :winner,    :through :matches
+  has_one    :loser,     :through :matches
+
```

Comment:
Consider adding: 
* Draft order - while this can be determined from replays, in the event of a lobby crash the draft order will be lost. Teams can be asked to track draft order for reliable stats.
* Replay URL or Replay ID - to retrieve the file itself or associated stats

Reply:
additional possible adds:
* Casters (user ids)
* Vod url (if casted, casters should have privileges to add the vod url for things the casted)

Reply:
I think `caster_id` ( scoped as a `User` ) and `vod_url` should instead live on the `Match` model, as those are almost always done on a match by match bases.

Reply:
Yeah, I've added a `caster` json to the match to house those details.

I could see an argument that the caster info could _technically_ differ between games. Although, we'd definitely want it on the match, which I'm leaning to just keeping it on the match.